### DISCUSSION TITLE
rerenderProps

### DISCUSSION THREAD

**Discussion by SurjanaJ** [img-mapper/img-mapper#71] [General]
hi, could you please provide an example on how to use the rerenderProps property?

**Comment by NishargShah** [comment]
`rerenderProps` is used when you want to rerender the whole component again based on the one prop you have changed. Basically, it is not required if you are using it with a controllable manner but if you using it as Uncontrollable manner, you required it. BTW, it will be removed in future versions if everything seems ok
